### ISSUE TITLE
Collapsible outline subheadings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#222]
```
What steps will reproduce the problem?
1.Load a PDF or DJVU document with an outline with subheadings
2.Bring up the outline view

What is the expected output?
Subheadings collapsed underneath the primary heading, so that you don't have to 
scroll through all the subheadings.  Similar to djview reader on windows/linux. 
 Option to tap on a drop-down arrow next to a main heading to unfold the 
subheadings underneath it.

What do you see instead?
All headings and subheadings listed making it hard to find what you want in 
long outlines.

What version of the product are you using?
v1.4.1

On what operating system?
Android 2.2

What device are you using?
Notion Ink Adam

Please provide any additional information below.

```

Original issue reported on code.google.com by `norri...@gmail.com` on 2 May 2012 at 1:35

**Comment by GoogleCodeExporter** [comment]
```
implemented in /trunk. will be available in 1.5 release. 
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 2 May 2012 at 6:08
- Changed state: **Fixed**
- Added labels: **Type-e**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 2 May 2012 at 6:09
- Added labels: **Type-Enhancement**
- Removed labels: **Type-e**
